src/components/cavatica/CavaticaCopyModal.js

```diff
@@ -208,9 +209,19 @@ const CavaticaCopyModal = ({
                 action: 'success',
                 component: SuccessToastComponent({ theme, selectedProjectData }),
               });
+              trackUserInteraction({
+                category: TRACKING_EVENTS.categories.fileRepo.actionsSidebar,
+                action: 'Copied Files to Cavatica Project',
+                label: uuids,
+              });
               props.onComplete();
             } catch (e) {
               //TODO: Display failure error.
+              trackUserInteraction({
+                category: TRACKING_EVENTS.categories.fileRepo.actionsSidebar,
+                action: 'Copied Files to Cavatica Project FAILED',
+                label: e,
```

Comment:
`e.message`, or stack trace? not sure if we want to jam in the whole error object here